README.md

```diff
@@ -25,12 +25,12 @@ steps:
   - uses: actions/checkout@v2
 
   - name: Auto Minify
-    uses: nizarmah/auto-minify@master
+    uses: nizarmah/auto-minify@v2.0
```

Comment:
Unless you have a specific reason to restrict it to `.0` I would suggest to just use `@v2`
(same for the other occurrences)

Reply:
@Spreeuw I do, I'm planning to release `v2.1` on Wednesday once #19 is merged.

Reply:
I'm not sure that answers my question 😉 
If you follow SemVer, `v2.1` would not have any breaking changes compared to `v2.0`, so then it would make sense to use `@v2` in your docs. If you **_do not_** follow SemVer and `v2.1` is a breaking change... Then you should indeed use `@v2.0` in your docs.

Reply:
Oh! @Spreeuw thanks for the clarification 👀 I've updated it to `@v2` since `v2.1` isn't a breaking change.

I'll read more about semantic versioning; I wasn't aware of it so thanks for pointing me in the right direction 😃